### PR TITLE
All about the store

### PR DESCRIPTION
**Pull request by kat-hi**
- include api version configuration
- new store concept: it is modularized (user, statistics and generator)
- jwt-handling is removed since backend uses "set-cookie" to save the jwt 

- there is now a cookie (because of FLASK-Sessions) that stores a signed and base64-encoded token that contains the username. I don't use it. But maybe I should do it..
- login and data requests are fixed, which is based on this dev-api-version: v1.1.0 (https://github.com/kubeportal/kubeportal-api-dev/pull/9)

### CODE REVIEW

File: src/api/backend.js [hunk 1]
```diff
@@ -7,8 +7,10 @@ function canReadURLFromEnv () {
 }
 
 export function setBaseURLWithDefaultOrEnvValue () {
-  const defaultUrl = 'https://kubeportal-dev.api.datexis.com/'
-  return canReadURLFromEnv() ? process.env['VUE_APP_BASE_URL'] : defaultUrl
+  const API_VERSION = 'v1.0.0'
```
Comment (troeger) [inline src/api/backend.js:10]: I assume you change this to 1.1.0 when everything is done.

Comment (kat-hi) [reply src/api/backend.js]: sure

**Comment by troeger** [comment]
I tried to test it by building the docker image and running it locally. The startup stopped here:

image

**Comment by kat-hi** [comment]
fixed

**Comment by troeger** [comment]
- [ ] Please stop performing a web pack run on every container startup. This belongs into the Dockerfile as part of the build phase.
- [ ] Mapping the container port outside (docker run -p 8080:8080) fails with a CORS error, since somehow port "8000" is epected. This should be documented, or the CORS check should work with 8080.
- [ ] Username / password login fails without an error notification.

grafik

**Comment by kat-hi** [comment]
I'll merge this PR and reopen a new one with your todos.

**Pull request merged**